playwright/playwright.config.ts

```diff
@@ -28,8 +28,8 @@ export default defineConfig({
   globalTimeout: 60 * 60 * 1000,
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
-  /* Retry on CI only */
-  retries: process.env.CI ? 1 : 0,
+  /* Retry once on CI */
+  retries: process.env.CI ? 1 : 2,
```

Comment:
Why 2 here?

Reply:
I have been having some issues with tests failing locally and I noticed that some of these test would fail around twice and then the third time would pass. The value is most definitely arbitrary as I could see it failing `n` times and the `n+1` time it passes.